imager/ImagerProfile/models.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import unicode_literals
+from django.db import models
+from django.contrib.auth.models import User
+from django.utils.encoding import python_2_unicode_compatible
+
+
+class ActiveImagerManager(models.Manager):
+    def get_queryset(self):
+        return super(ActiveImagerManager,
+                     self).get_queryset().filter(user__is_active=True)
+
+
+@python_2_unicode_compatible
+class ImagerProfile(models.Model):
+    """Imager profile that is attached to user on creation"""
+    user = models.OneToOneField(User, related_name='profile')
+
+    profile_picture = models.ImageField(null=True, blank=True,
+                                        upload_to='images')
+
+    phone_number = models.CharField(max_length=20, blank=True)
+
+    birthday = models.DateField(null=True, blank=True)
+
+    phone_privacy = models.BooleanField(default=False)
+    birthday_privacy = models.BooleanField(default=False)
+    picture_privacy = models.BooleanField(default=False)
+    name_privacy = models.BooleanField(default=False)
+    email_privacy = models.BooleanField(default=False)
+
+    objects = models.Manager()
+    active = ActiveImagerManager()
+
+    following = models.ManyToManyField('self',
+                                       symmetrical=False,
+                                       null=True,
+                                       blank=True,
+                                       related_name='followers')
+    blocking = models.ManyToManyField('self',
+                                      symmetrical=False,
+                                      null=True,
+                                      blank=True,
+                                      related_name='+')
+
+    def follow(self, imagerprofile):
+        if self in imagerprofile.blocking.all():
+            return 'User has blocked you'
+        if self is imagerprofile:
+            return 'You cannot follow yourself'
+        self.following.add(imagerprofile)
```

Comment:
As always, it is better to raise an error than to return a string for problems like this.  You can react to that error in the calling context by sending a text message to the user, but the method should raise an error and then allow the calling context to decide what to do with it.